_posts/2020-08-31-rico-logging.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  'Rico logging'
```

Comment:
Maybe a more catchy headline? Or with more information? Idea: "Centralized and contextual logging with Rico"